coding/multilang_utf8_string.hpp

```diff
@@ -75,15 +75,8 @@ class StringUtf8Multilang
   /// @returns empty string if langCode is invalid.
   static char const * GetTransliteratorIdByCode(int8_t langCode);
 
-  inline bool operator== (StringUtf8Multilang const & rhs) const
-  {
-    return (m_s == rhs.m_s);
-  }
-
-  inline bool operator!= (StringUtf8Multilang const & rhs) const
-  {
-    return !(*this == rhs);
-  }
+  inline bool operator==(StringUtf8Multilang const & rhs) const { return (m_s == rhs.m_s); }
```

Comment:
Please drop the parentheses while you're here.

Reply:
Done